docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+---
+title: Develop & Deploy Your Own React Monorepo App in Under 2 Hours, Using Lerna, Travis & Now
```

Comment:
```suggestion
 title: Publish Multiple Gatsby Sites in a Monorepo Using Lerna, Travis & Now
```

Reply:
I'd like to keep the keywords Develop and Deploy in the title, as it describes the content of my article better, referencing the way in which it's structured. 'Publish Multiple Gatsby Sites' doesn't capture it as much. Saying that, I'm happy to replace 'React' with 'Gatsby' as a means of emphasising our favourite highly performant site generator

Reply:
Title should be more descriptive and a hook to understand what you would do at a glance, and "Publish Multiple Gatsby Sites" is more descriptive than "Develop & Deploy Your Own React[/Gatsby] Monorepo App". 

We don't use the language "Gatsby Monorepo App" anywhere else, and it's not clear immediately what this means and raises additional Qs. Are all Gatsby apps monorepos? Etc.